docs/guides/docs/running-a-node/running-a-beta-4-node.mdx

```diff
@@ -0,0 +1,304 @@
+---
+title: Running a Beta-4 Node
+category: Running a Node
+parent:
+  label: Guides
+  link: /guides
+---
+
+# Running a local Fuel node connected to Beta-4 using P2P
+
+## Installation
+Make sure you have the latest version of fuelup by running the following command:
+
+```sh
+fuelup self update
+```
+
+Then run `fuelup toolchain install beta-4` to install the `beta-4` toolchain.
```

Comment:
I'm trying these commands myself, but at this step I'm getting:
```
Invalid toolchain metadata within input 'beta-4' - Failed to parse date or target
```

Reply:
The current installation is `fuelup toolchain install beta-4-rc.2` but will be `fuelup toolchain install beta-4` in anticipation of the public release.